common/src/main/java/com/microsoft/identity/common/internal/dto/PrimaryRefreshTokenRecord.java

```diff
@@ -0,0 +1,203 @@
+
+        /**
+         * String of prt_protocol_version.
+         */
+        public static final String PRT_PROTOCOL_VERSION = "prt_protocol_version";
+    }
+
+    /**
+     * 1st Party Application Family ID.
+     */
+    @SerializedName(FAMILY_ID)
+    private String mFamilyId;
+
+    /**
+     * PRT expiry time. This value is returned from the server as refresh_token_expires_in that
+     * should be added to local time.
+     */
+    @SerializedName(EXPIRES_ON)
+    private String mExpiresOn;
+
+    /**
+     * Session key of PRT.
+     */
+    @SerializedName(SESSION_KEY)
+    private String mSessionKey;
+
+    /**
+     * The version of the PRT Protocol being used, version 3 supports no device_id.
+     */
+    @SerializedName(PRT_PROTOCOL_VERSION)
+    private String mPrtProtocolVersion;
+
+    /**
+     * Session key expiry time. This value is determined by the client and set to 60 days after the
+     * session key was initally issued
+     */
+    @SerializedName(SESSION_KEY_ROLLING_DATE)
+    private String mSessionKeyRollingDate;
+
+    /**
+     * Gets the family_id.
+     *
+     * @return The family_id to get.
+     */
+    public String getFamilyId() {
+        return mFamilyId;
+    }
+
+    /**
+     * Sets the family_id.
+     *
+     * @param familyId The family_id to set.
+     */
+    public void setFamilyId(final String familyId) {
+        mFamilyId = familyId;
+    }
+
+    public boolean isExpired(final String expirationTimeEpochSeconds) {
+        return Long.parseLong(expirationTimeEpochSeconds) * 1000 < System.currentTimeMillis();
+    }
+
+    @Override
+    public boolean isExpired() {
+        return isExpired(getExpiresOn());
+    }
+
+    /**
+     * Gets the expires_on.
+     * @return The expires_on to get.
+     */
+    public String getExpiresOn() {
+        return mExpiresOn;
+    }
+
+    /**
+     * Sets the expires_on.
+     * @param expiresOn The expires_on to set.
+     */
+    public void setExpiresOn(String expiresOn) {
+        mExpiresOn = expiresOn;
+    }
+
+    /**
+     * Gets the session_key.
+     * @return The session_key to get.
+     */
+    public String getSessionKey() {
+        return mSessionKey;
+    }
+
+    /**
+     * Sets the session_key.
+     *
+     * @param sessionKey The session_key to set.
+     */
+    public void setSessionKey(String sessionKey) {
+        mSessionKey = sessionKey;
+    }
+
+    /**
+     * Gets the prt_protocol_version.
+     *
+     * @return The prt_protocol_version to get.
+     */
+    public String getPrtProtocolVersion() {
+        return mPrtProtocolVersion;
+    }
+
+    /**
+     * Sets the prt_protocol_version.
+     *
+     * @param prtProtocolVersion The prt_protocol_version to set.
+     */
+    public void setPrtProtocolVersion(String prtProtocolVersion) {
+        mPrtProtocolVersion = prtProtocolVersion;
+    }
+
+    /**
+     * Gets the session_key_rolling_date.
```

Comment:
Let's include the format of this value. I'm guessing it's probably epoch seconds? Would be good to have this info in the javadoc

Reply:
Thanks, Comments and format updated.